pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TransactionEndToEndTest.java

```diff
@@ -1223,6 +1223,8 @@ public void testSendTxnAckMessageToDLQ() throws Exception {
         // the message will be sent to DLQ, can't receive
         assertNull(consumer.receive(3, TimeUnit.SECONDS));
 
+        assertEquals(3, ((ConsumerImpl<?>) consumer).getAvailablePermits());
```

Comment:
```suggestion
        assertEquals(((ConsumerImpl<?>) consumer).getAvailablePermits(), 3);
```

The second param is the expected value. We will get counterintuitive information if the test fails here.